geonode/maps/templates/maps/map_detail.html

```diff
@@ -207,7 +207,7 @@ <h4 class="list-group-item-heading">{% trans "Map Layers" %}</h4>
         <li class="list-group-item">
           <h4>{% trans "Copy this map" %}</h4>
           <p>{% trans "Duplicate this map and modify it for your own purposes" %}</p>
-          <a href="{% url "new_map" %}?copy={{ resource.id }}" class="btn btn-default btn-md btn-block">{% trans "Create a New Map" %}</a>
+          <a href="{% url "copy_map" resource.id %}" class="btn btn-default btn-md btn-block">{% trans "Create a New Map" %}</a>
```

Comment:
the link is broken for QGIS Server only, but it works with geoserver.
You can check it yourself in staginggs.geonode.kartoza.com

The handler for the url is in here:

https://github.com/kartoza/geonode/blob/master-qgis_server/geonode/maps/urls.py#L34

As you can see, the view function was branched according to ogc backend currently being used. It works with geoserver because it was using `new_map_view` which points to `new_map` function. However for qgis_server, it points to `MapCreateView.as_view()`. So, the previous state: it works in Geoserver but not in qgis_server (because Ismail haven't made that yet). And if you change it like this, it will work for qgis_server but not for geoserver, because your code didn't branch and you didn't handle geoserver case.

But I've seen your new changes. You need to make sure it works with geoserver before you merge it. I will rereview this PR.